app/views/catalog/_featured_content.html.erb

```diff
@@ -1,29 +1,34 @@
-<div class="col-md-12">
-  <section class="topics">
-    <div class="line-behind-text">
-      <h6 class="er_toc_tag" id="er-toc-id-2"><%= t('pulmap.search.topics.heading') %></h6>
-    </div>
-    <div class="grid-items">
-      <a href="/?f%5Blayer_geom_type_s%5D%5B%5D=Image&q=Sanborn" class="grid-item">
-        <span class="icon-building"></span>
-        <h1>Sanborn</h1>
-        <p>Detailed fire insurance maps of cities and towns in New Jersey.</p>
-      </a>
-      <a href="/?f%5Bdc_format_s%5D%5B%5D=Shapefile&q=parcels" class="grid-item">
-        <span class="icon-locationmap"></span>
-        <h1>Parcels</h1>
-        <p>Property boundry data for identification of ownership, land use, and land value.</p>
-      </a>
-      <a href="/?f%5Bdc_subject_sm%5D%5B%5D=population" class="grid-item">
-        <span class="icon-graphbar"></span>
-        <h1>Census</h1>
-        <p>Worldwide population and demographic data.</p>
-      </a>
-        <a href="/?utf8=✓&search_field=all_fields&sort=timestamp+desc&q=" class="grid-item">
-        <span class="icon-map"></span>
-        <h1>Recent Additions</h1>
-        <p>Recent additions to the collection.</p>
-      </a>
-    </div>
-  </section>
+<div class="intro col-md-12">
+    <section class="topics">
+      <h1><small>Introducing a new research tool to </small><br/>
+        Explore Maps and Geospatial Data</h1>
+
+      <div class="grid-items">
+        <a href="/?utf8=%E2%9C%93&search_field=all_fields&q=" class="grid-item">
+          <span class="icon-locationmap"></span>
+          <h1>Browse Collection</h1>
```

Comment:
The text specified in the requested change

Reply:
Done!